<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>abonentmncfg</class>
 <widget class="QDialog" name="abonentmncfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Список абонентов</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Канал 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t101">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t102">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t103">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t104">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t105">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t106">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t107">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t108">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t109">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Канал 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t201">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t202">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t203">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t204">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t205">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t206">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t207">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t208">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t209">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string>9999</string>
           </property>
           <property name="text">
            <string>0000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
